comparisons/navigation/tabs/a11y.txt

```diff
@@ -0,0 +1 @@
+Usage:
```

Comment:
Similar to accordion, lots of keyboard and ARIA support needed https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel